app/assets/stylesheets/components/overview.scss

```diff
@@ -13,3 +13,12 @@
 .summary-link-colour:hover {
   color: govuk-colour("white")
 }
+
+.overview-section-allignment{
+  padding-left: 0px;
+  padding-right: 0px;
+}
+
+.activity-section-allignment{
+  margin-left: -15px;
```

Comment:
I'd be surprised if this works on all screen sizes/devices. 

Is there any way for us to achieve this without resorting to pixel-based styling?

Reply:
It works on the multiple smaller displays, I tested it ensuring it was responsive